appointment/views.py

```diff
@@ -96,11 +96,10 @@ def get_available_slots_ajax(request):
 
     # Check if the selected_date is today and filter out past slots
     if selected_date == date.today():
-        # Get the current time in EDT timezone
-        current_time_edt = datetime.now(pytz.timezone(settings.TIME_ZONE)).time()
```

Comment:
codecov said this line is not covered, but it's not a new line, so wasn't covered before, right?

Reply:
Not sure if it's this line, since code has been added in previous lines, it may have been moved. Not sure codecov is reliable here.